public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -15,9 +15,11 @@ import { DashboardModel } from '../../state';
 import { KioskMode } from 'app/types';
 import { ShareModal } from 'app/features/dashboard/components/ShareModal';
 import { SaveDashboardModalProxy } from 'app/features/dashboard/components/SaveDashboard/SaveDashboardModalProxy';
+import { DashboardChatModalProxy } from 'app/features/dashboard/components/DashChat/DashChatModalProxy';
```

Comment:
Do we need the proxy component in this case? I think we are doing some additional logic for `SaveDashboardModalProxy` that don't apply here

Reply:
Works without it, removed